warp10/src/main/java/io/warp10/standalone/InMemoryChunkSet.java

```diff
@@ -266,7 +272,7 @@ public GTSEncoder fetchEncoder(long now, long timespan, int preBoundary, int pos
     int nowchunk = chunk(now) + this.chunkcount;
     
     // Compute the first timestamp (included)
-    long firstTimestamp = now - timespan + 1;
+    long firstTimestamp = then;
```

Comment:
Not sure that `then` should be included, still that convention problem.

Reply:
It should, see above.